ext/com_dotnet/com_variant.c

```diff
@@ -975,6 +975,7 @@ PHP_FUNCTION(variant_date_from_timestamp)
 
 	/* Invalid after 23:59:59, December 31, 3000, UTC */
 	if (!tmv) {
+		/* TODO Promote to ValueError? */
 		php_error_docref(NULL, E_WARNING, "Invalid timestamp " ZEND_LONG_FMT, timestamp);
```

Comment:
I think we can promote this value error (the valid range of timestamps need to be documented, but that should happen anyway).